### DISCUSSION TITLE
Remove request lists

### DISCUSSION THREAD

**Discussion by freearhey** [iptv-org/community#521] [Ideas]
I propose to close all request lists in favor of single-channel requests. 

As practice has shown, these lists are very difficult to keep up to date and they only grow endlessly. Unfortunately, we can not guarantee that the channel added to the playlist today will not fail tomorrow and the list will need to be updated again. This leads to even more confusion. Ordinary requests at least can be closed automatically via pull request.

![image](https://user-images.githubusercontent.com/7253922/137828107-91b2b91e-411c-48f0-97ec-54f979788652.png)

All requests previously placed in the lists can be reopened (at least if they contain only one channel). Otherwise new requests must be created.

Filtering queries by country can be done through labels.

**Comment by BellezaEmporium** [comment]
LGTM. It gets difficult over time, and if we add one channel that falls in one of the requests, we might often forget that it was somewhere inside those lists, and it creates even more confusion (it's been added, but not notified ?)

**Comment by Dum4G** [comment]
@freearhey I think we can have a practice of closing stale requests via bot then. The fact people expect miracles from us doesn't necessarily mean we'll ever have a stable source.

**Reply by freearhey** [reply]
I personally do not see the point of closing requests that are not fulfilled. If we can't do it now, it doesn't mean that someone else can't do it in the future. And you don't have to respond to these requests, you can just skip them.
